USAGE.md

```diff
@@ -89,7 +89,7 @@ Below is a table of available configuration parameters followed by detailed desc
 | manta.no_auth                      | MANTA_NO_AUTH                  | false                                | Y*                       |
 | manta.disable_native_sigs          | MANTA_NO_NATIVE_SIGS           | false                                | Y*                       |
 | manta.verify_uploads               | MANTA_VERIFY_UPLOADS           | true                                 | Y                        |
-| manta.timeout                      | MANTA_TIMEOUT                  | 20000                                |                          |
```

Comment:
I arbitrarily choose 2 seconds for the connection timeout. @nairashwin952013 can you ask around and see what other people think is a reasonable default for a client connection timeout to Manta?

Reply:
Yes, I wanted to verify too if 2s is an optimal ```default``` Timeout setting for the java-manta SDK ?